OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -54,6 +58,26 @@ public async Task<IActionResult> ChangeEmail(ChangeEmailViewModel model)
         {
             logger.LogDebug($"{path} started. User(id): {userId}.");
 
+            if (model.Submit == localizer["Cancel"])
```

Comment:
Please clarify why this check is done before validation of `ModelState.IsValid` (`if` statement below). Shouldn't it be reordered?

Reply:
Thank you, it's clear now